identity-service/src/audiusLibsInstance.js

```diff
@@ -35,4 +33,16 @@ async function initAudiusLibs () {
   return audiusInstance
 }
 
-module.exports = initAudiusLibs
+class AudiusLibsWrapper {
+  constructor () {
+    this.audiusLibsInstance = null
+  }
+
+  async init () {
+    this.audiusLibsInstance = await initAudiusLibs()
+  }
```

Comment:
there should be a getAudiusLibs getter function that everything calls. we shouldn't be calling class variables directly

Reply:
I see, that makes sense, thanks.